### PR TITLE
[SKIP CI] .github/daily-tests.yml: add sparse-zephyr.yml

### PR DESCRIPTION
**Pull request by marc-hb**
Because why not.

### CODE REVIEW

**Comment by marc-hb** [comment]
In https://github.com/thesofproject/sof/actions/runs/4318983392/jobs/7537920586 checkpatch is clueless and wrong

File: .github/workflows/daily-tests.yml [hunk 1]
```diff
@@ -18,3 +18,5 @@ jobs:
     uses: ./.github/workflows/pull-request.yml
   run-zephyr-builds:
     uses: ./.github/workflows/zephyr.yml
+  sparse-zephyr:
```
Comment (kv2019i) [inline .github/workflows/daily-tests.yml:21]: I'd argue some better rationale could be added to the git commit that "why not".

**Review by kv2019i — APPROVED** [review]
Makes sense to add this as well, but I'd still add a bit of background to the git commit message.

File: .github/workflows/daily-tests.yml (hunk 1 above)
Comment (marc-hb) [reply .github/workflows/daily-tests.yml:21]: Fair enough, I should probably have said something better like "there is no reason to exclude sparse from daily tests" :-)

Thanks for not blocking this.

**Review by lgirdwood — APPROVED** [review]

**Comment by aborisovich** [comment]
Because that's why:
image

Please hotfix.

**Comment by marc-hb** [comment]
Thanks for spotting this, tentative fix submitted in:
- #7211

**Pull request merged**